Automattic/wp-calypso
# Stats: Replace the use of the PostStatsStore by the redux approach (query + selector)

I replaced the use of the PostStatsStore by the redux approach (query + selector)
I also dropped the store entirely (not used anymore) and added a heartbeat prop the QueryPostStat component to refresh the results each n milliseconds

**Testing Instructions**

No functional changes.

 * Navigate to my sites blog posts page
 * Check that the posts' view count is displayed correctly

cc @timmyc @mdawaffe
